template.yml

```diff
@@ -40,62 +49,65 @@ Resources:
       CodeUri: ./backend-lambda
       Environment:
         Variables:
-          DATABASE_URL: !Ref DATABASE_URL
+          DATABASE_URL: !Ref DatabaseUrl
       Events:
```

Comment:
Do we still need this block if we have the RestApi block?

Reply:
We do not need it, removing.